Before: b9cd900bf090b58c8c6d308ff87f21546a770c01df925d6636fae811ba5d3b92
After: 731bfb1267ae6acb243bea5c9fd6bd65f6352703fc101c3c0a1a24862e7cc0ea
Patch: a7beefc69b9678f7f915debb42639211fb4db2c7d6171bec114076ee49e77e5a
--- bin/%plugin_name%.tt	2012-08-15 12:04:48.930195662 +1000
+++ /dev/null	2012-08-14 08:14:42.724947336 +1000
@@ -1,21 +0,0 @@
-#!/usr/bin/env ruby
-# This file is generated by Corvid.
-
-# Determine project root
-require 'pathname'
-project_root= File.expand_path('../..', Pathname.new(__FILE__).realpath)
-
-# Load Bundler
-ENV['BUNDLE_GEMFILE'] ||= (
-  gemfile= File.join(project_root, 'Gemfile')
-  File.exists?(gemfile) && gemfile
-)
-require 'rubygems'
-require 'bundler/setup' if ENV['BUNDLE_GEMFILE']
-
-# Start CLI
-require 'corvid/plugin_cli'
-Corvid::PluginCli.start project_root:        project_root,
-                        plugin_require_path: '<%= require_path %>',
-                        plugin_class_name:   '<%= class_name %>'
-
